# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/python/hotfix-ncursesw.patch
# Copyright (C) 2023 - 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

./Modules/_cursesmodule.c:763:9: error: implicit declaration of function 'setcchar'; did you mean 'getwchar'? [-Werror=implicit-function-declaration]
  763 |         setcchar(&wcval, wstr, attr, PAIR_NUMBER(attr), NULL);
      |         ^~~~~~~~
      |         getwchar
...

--- Python-3.12.0/Include/py_curses.h.vanilla	2023-10-09 15:38:40.115025579 +0200
+++ Python-3.12.0/Include/py_curses.h	2023-10-09 15:39:15.071025694 +0200
@@ -31,7 +31,7 @@
 #endif
 
 #ifdef HAVE_NCURSES_H
-#include <ncurses.h>
+#include <ncursesw/ncurses.h>
 #else
 #include <curses.h>
 #endif
